sdk/program/src/vote/state/mod.rs

```diff
@@ -221,16 +221,21 @@ impl VoteStateUpdate {
 
 #[cfg_attr(
     feature = "frozen-abi",
-    frozen_abi(digest = "5PFw9pyF1UG1DXVsw7gpjHegNyRycAAxWf2GA9wUXPs5"),
+    frozen_abi(digest = "DqMoZ6hSsG1wguKx49kjZmU8sEu7jjzDWqg14qxoeinM"),
     derive(AbiExample)
 )]
 #[derive(Serialize, Default, Deserialize, Debug, PartialEq, Eq, Clone)]
 pub struct TowerSync {
-    /// The proposed tower
+    /// the proposed tower
     pub lockouts: VecDeque<Lockout>,
-    /// The proposed root
+    /// the proposed root
     pub root: Option<Slot>,
-    /// signature of the bank's state at the last slot
+    /// signature of the VoteState's at the latest slot in `lockouts`
+    pub yolo_hash: Hash,
```

Comment:
Is it possible to call this ephemeral_hash? Because we might use yolo_hash later for other things.

Reply:
i'm open to naming suggestions, i used ephemeral in the title but i find it a little confusing. 
There's nothing about the hash that is ephemeral, it's still stored permanently in the votes and finalized blocks. Similarly the vote states aren't ephemeral either since they will eventually be committed.

Reply:
I don't have strong preference on names, we can ask Toly since it's his idea.

Reply:
optimistic_hash?
This is the hash of the votes that we, optimistically assumed could pay fees, right?

Reply:
I do like optimistic, it makes sense but I think it might be confusing with optimistic confirmation. there already is such a thing as the OC hash 😅.

maybe just something simple like vote_hash? vote_state_hash?

Reply:
Or maybe fast_hash?